COMETwebapp/COMETwebapp/Program.cs

```diff
@@ -0,0 +1,11 @@
+using COMETwebapp;
```

Comment:
Please add File header. Maybe discuss with @samatrhea what it should look like in Blazor/Razor.